content/en/docs/tutorials/kubernetes-basics/create-cluster/cluster-intro.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Using Minikube to Create a Cluster
+weight: 10
+---
+
+## {{% heading "objectives" %}}
+
+* Learn what a Kubernetes cluster is.
+* Learn what Minikube is.
+* Start a Kubernetes cluster on your computer.
+
+## Kubernetes Clusters
+
+<div class="col-md-8">
+  <div class="content__box content__box_fill">
+    <p><i>Kubernetes is a production-grade, open-source platform that orchestrates
+      the placement (scheduling) and execution of application containers
+      within and across computer clusters.</i></p>
+  </div>
+</div>
```

Comment:
I would either use this:

```suggestion
{{% alert title="What is Kubernetes" color="primary" %}}
      Kubernetes is a production-grade, open-source platform that orchestrates
      the placement (scheduling) and execution of application containers
      within and across computer clusters.
{{% /alert %}}
```
_or_ make a similar but more custom shortcode

_or_ a plain paragraph with a custom class (and some associated CSS).

Reply:
I've used the `{{% alert %}}` suggestion.